src/interface/sparse_pack.hpp

```diff
@@ -333,6 +333,19 @@ class SparsePack : public SparsePackBase {
   }
 
   // flux() overloads
+  template <class... Args>
+  KOKKOS_INLINE_FUNCTION auto &flux(const int b, const TopologicalElement te,
+                                    Args &&...args) const {
+    const int dir = (static_cast<int>(te) % 3) + 1;
+    return flux(b, dir, std::forward<Args>(args)...);
+  }
+
+  template <class... Args>
+  KOKKOS_INLINE_FUNCTION auto &flux(const TopologicalElement te, Args &&...args) const {
+    const int dir = (static_cast<int>(te) % 3) + 1;
+    return flux(dir, std::forward<Args>(args)...);
+  }
```

Comment:
Did these not already exist? What's different here?

Reply:
That they take a `TopologicalElement` argument instead of a `CoordinateDirection`. This exists for `operator()` but not for `SparsePack::flux(...)`